core/player/src/view/view.ts

```diff
@@ -94,21 +96,29 @@ export class ViewInstance implements ValidationProvider {
   // TODO might want to add a version/timestamp to this to compare updates
   public lastUpdate: Record<string, any> | undefined;
 
-  constructor(initialView: ViewType, resolverOptions: Resolve.ResolverOptions) {
+  constructor(initialView: ViewType, resolverOptions: Resolve.ResolverOptions & ViewControllerOptions) {
     this.initialView = initialView;
     this.resolverOptions = resolverOptions;
     this.hooks.onTemplatePluginCreated.tap("view", (templatePlugin) => {
       this.templatePlugin = templatePlugin;
     });
+    resolverOptions.flowController.current?.hooks.transition.tap("view", (state1, state2) => {
+      this.transitioning = true;
+    })
+    resolverOptions.flowController.current?.hooks.afterTransition.tap("view", (flowInstance) => {
+      this.transitioning = false;
+    })
```

Comment:
You'll want to use the `.flow` hook on the flow-controller instead of using `current` directly.

Reply:
while accessible. the call of that hook takes place earlier than the tap due to flow/view scope diff, so I moved the logic out to the view controller